src/cascadia/TerminalSettingsModel/ApplicationState.h

```diff
@@ -21,11 +22,21 @@ Module Name:
 // This macro generates all getters and setters for ApplicationState.
 // It provides X with the following arguments:
 //   (type, function name, JSON key, ...variadic construction arguments)
-#define MTSM_APPLICATION_STATE_FIELDS(X) \
-    X(std::unordered_set<winrt::guid>, GeneratedProfiles, "generatedProfiles")
+#define MTSM_APPLICATION_STATE_FIELDS(X)                                       \
+    X(std::unordered_set<winrt::guid>, GeneratedProfiles, "generatedProfiles") \
+    X(Windows::Foundation::Collections::IVector<winrt::Microsoft::Terminal::Settings::Model::WindowLayout>, PersistedWindowLayouts, "persistedWindowLayouts")

```

Comment:
If we move this down into the namespace below, it'll be a clearer signal that symbols in the namespace scope can be used without full specifications.
Because `winrt::Microsoft::Terminal::Settings::Model::WindowLayout` can be shortened to `Model::WindowLayout`.